### PR TITLE
chore(codeowners): map performance dir to web team

### PR DESCRIPTION
**Pull request by loksonarius**
Who gets pinged for reviews in the `performance` directory?

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -4,10 +4,11 @@
...
 # team-specific repo segments
-/.github  @determined-ai/infrastructure
-/master   @determined-ai/backend
-/agent    @determined-ai/backend
-/proto    @determined-ai/backend
+/.github        @determined-ai/infrastructure
+/master         @determined-ai/backend
+/agent          @determined-ai/backend
+/proto          @determined-ai/backend
+/performance    @determined-ai/web
```
Comment (stoksc) [inline .github/CODEOWNERS:11]: maybe group entries by team?

**Review by stoksc — APPROVED** [review]
lgtm.

**Review by tara-det-ai — COMMENTED** [review]
nothing to review

**Comment by rb-determined-ai** [comment]
Maybe I don't understand, but why is ml-sys being requested for review here?  I don't see any files that we own being modified.

**Comment by ioga** [comment]
@rb-determined-ai it was caused by Carolina's force push to the main branch.

**Comment by rb-determined-ai** [comment]
ah, thank you

File: .github/CODEOWNERS (hunk 1 above)
Comment (dannysauer) [reply .github/CODEOWNERS:11]: Aren't they already grouped?  Or do you mean put all the directories on one comma separated line?

**Pull request merged**